exosphere-shared/example-apps/crashing-service/application.yml

```diff
@@ -6,5 +6,7 @@ services:
   public:
     crasher:
       location: ./crasher
+      docker_image: exospheredev/test-service-that-crashes-on-startup
```

Comment:
Just out of curiosity, aren't we also specifying the docker image in `service.yml`? Given that, why is it necessary to specify them here as well? Can't we do either `location` that points to a directory with `service.yml` that contains the Docker image, or if one doesn't have the source code checked out, then they need to provide a `docker_image` key here. But not both.